CONTRIBUTING.md

```diff
@@ -0,0 +1,70 @@
+# Contributing to Asgardeo Mobile SDKs
+
+We would love for you to contribute to our mobile SDKs and help make it even better than it is today!
+As a contributor, here are the guidelines we would like you to follow:
+
+- [Commit Message Guidelines](#commit-message-guidelines)
+
+## Commit Message Guidelines
+
+*This specification is inspired by and supersedes the [AngularJS commit message format][commit-message-format].*
+
+We have very precise rules over how our Git commit messages must be formatted.
+This format leads to **easier to read commit history**.
+
+Each commit message consists of a **header**, a **body**, and a **footer**.
+
+```
+<header>
+<BLANK LINE>
+<body>
+<BLANK LINE>
+<footer>
+```
+
+The `header` is mandatory and must conform to the [Commit Message Header](#commit-header) format.
+
+The `body` is mandatory for all commits except for those of type "docs".
+When the body is present it must be at least 20 characters long and must conform to the [Commit Message Body](#commit-body) format.
+
+The `footer` is optional. The [Commit Message Footer](#commit-footer) format describes what the footer is used for and the structure it must have.
+
+
+#### <a name="commit-header"></a>Commit Message Header
+
+```
+<type>(<scope>): <short summary>
+  │       │             │
+  │       │             └─⫸ Summary in present tense. Not capitalized. No period at the end.
+  │       │
+  │       └─⫸ Commit Scope: android|android-core
+  │
+  └─⫸ Commit Type: build|ci|docs|feat|fix|perf|refactor|chore|test
+```
+
+The `<type>` and `<summary>` fields are mandatory, the `(<scope>)` field is optional.
+
+
+##### Type
+
+Must be one of the following:
+
+* **build**: Changes that affect the build system or external dependencies (example scopes: gulp, broccoli, npm)
+* **ci**: Changes to our CI configuration files and scripts (examples: CircleCi, SauceLabs)
+* **docs**: Documentation only changes
+* **feat**: A new feature
+* **fix**: A bug fix
+* **perf**: A code change that improves performance
+* **refactor**: A code change that neither fixes a bug nor adds a feature
+* **chore**: Housekeeping tasks that doesn't require to be highlighted in the changelog
+* **test**: Adding missing tests or correcting existing tests
+
+
+##### Scope
+The scope should be the name of the npm package affected (as perceived by the person reading the changelog generated from commit messages).
+
+The following is the list of supported scopes:
+
+* `android` - Changes to the main `android` package.
+* `android-core` - Changes to the `android-core` package.
+* `docs` - Changes to the documentation files.
```

Comment:
Isn't `doc` scope already covered by type `docs`? Having a scope named `docs` result in commit messages such as `docs(docs)` which seems redundant.

Reply:
ack, lets remove the docs type

Reply:
Fixed with https://github.com/asgardeo/mobile-ui-sdks/pull/1/commits/67ea36cc9fef864e59221f887574afe50546eea3